README.md

```diff
@@ -1 +1,28 @@
-# hw01
\ No newline at end of file
+# hw01
+# My Introduction
+
+![Picture of me from the summer](https://scontent-ort2-1.xx.fbcdn.net/v/t1.0-9/68551018_2406605662727345_5013656229612879872_o.jpg?_nc_cat=101&_nc_sid=8024bb&_nc_ohc=h5MsPZwZ3-8AX_vzpJu&_nc_ht=scontent-ort2-1.xx&oh=ccf4c5bde72644f8030412a87d017d4a&oe=5EBBAFB4)
```

Comment:
Ah, nice incorporation of a picture here! That's so awesome that you got to speak at a Bernie event! I could not figure out how to attach a jpg but now I have this to reference to.